core-services/prow/02_config/_labels.yaml

```diff
@@ -661,6 +661,262 @@ repos:
         target: both
         addedBy: label
 
+  openshift/prometheus
```

Comment:
This is not valid YAML

Reply:
that's what i get for trying to script this.  fixed (along w/ the others), thanks